docs/reference-implementation/Getting-Started.md

```diff
@@ -14,6 +14,11 @@ This guide describes the end-to-end process for setting up all pre-requisites an
 
 ## Pre-requisites
 
+The Azure Mission-Critical reference implementation gets deployed into an Azure Subscription. For this you need to have either:
+
+- **Owner permissions** yourself and to be able to create a Service Principal which you are able to grant Owner as well, or
```

Comment:
Owner permission doesn't allow the user to create an SPN. Can we separate it and we can also mention that User Access Administrator (UAA) would also be enough to make a role assignment for the SPN

Reply:
hmm good point. What permissions do you need to create an SPN?

Reply:
Aren't these the two viable options we have here:

a) use a pre-provisioned service principals with subscription-level owner permissions 
b) auto-create a spn as part of the creation of a service connection in azure devops (what permissions does that require)?

Option a) implies that you have to get one (or create one yourself) 

We're not global administrators in the microsoft tenant, but we can use the auto create option in ADO, can't we? And we can also manually create SPs.

Reply:
App registrations in the User Settings on the tenant
![image](https://user-images.githubusercontent.com/12544697/157452491-a3cb2cca-e39e-48bb-8c00-e470293a752d.png)

Reply:
how about now?

Reply:
added a minor change